mhkit/dolfyn/velocity.py

```diff
@@ -938,49 +714,6 @@ def turbulent_kinetic_energy(self, veldat, noise=[0, 0, 0], detrend=True):
 
         return da
 
-    def stresses(self, veldat, detrend=True):
```

Comment:
Was this function moved/renamed, or removed entirely?

Reply:
I renamed to `reynolds_stress` to make it more descriptive and moved to the ADV/turbulence.py file because this algorithm (simply the covariance between velocity signals) is only accurate for point-measurements (ADV, shear probe, etc).